internal/jobcontainers/process.go

```diff
@@ -217,7 +235,9 @@ func signalProcess(pid uint32, signal int) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to open process")
 	}
-	defer windows.Close(hProc)
+	defer func() {
```

Comment:
These could just be left as the original defer windows.Close(hproc) with no functionality change.

Reply:
I think our linter was whining about it as we weren't handling the error, but begs the question of how it didn't whine the first time..

Reply:
Let me look what we can get away with

Reply:
internal\jobcontainers\process.go:252:27: Error return value of `windows.FreeLibrary` is not checked (errcheck)
        defer windows.FreeLibrary(k32)
        
Seems to only care about the freelibrary for some odd reason